docs/reference/configuration.txt

```diff
@@ -254,6 +254,12 @@ for details on driver options.
 
     # Configure Mongoid-specific options. (optional)
     options:
+      # When chaining the same operators that use the same field, setting this
+      # feature flag to true will cause those operators to be combined using an
+      # and. Setting this feature flag to false will cause the later chained
+      # operators to overwrite the earlier ones. (default: false)
+      #and_chained_operators: true
```

Comment:
I would rename the option to `overwrite_chained_operators` and invert it to make it be in line with how other options behave (the false value is default behavior of 8.0+).